plugin/pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -28,6 +28,26 @@ const (
 	// and removed when node becomes reachable (NodeReady status ConditionTrue).
 	TaintNodeUnreachable = "node.alpha.kubernetes.io/unreachable"
 
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeOutOfDisk would be automatically added by node controller
+	// when node becomes out of disk, and removed when node has enough disk.
+	TaintNodeOutOfDisk = "node.kubernetes.io/outOfDisk"
```

Comment:
see the discussion on including alpha in the name in https://github.com/kubernetes/community/pull/819#discussion_r128887276

tl;dr: if you include alpha in the name, there is no reasonable way to migrate people to the beta and ga versions. feature-gating the alpha function is sufficient to keep it from affecting anything until it graduates to beta or an admin opts in to using it in alpha mode.